content/projects/gsoc/2019/project-ideas/automatic-spec-generator-for-jenkins-rest-api.adoc

```diff
@@ -0,0 +1,22 @@
+---
+layout: gsocprojectidea
+title: "Automatic Specification Generator for Jenkins REST API"
+goal: "Find and implement the extraction of the REST APIs from the sources and generate and publish the REST APIs respective documentation"
+category: Plugins
+year: 2019
+status: draft
+skills:
+- Java
+- CSS
+- HTML
+mentors:
+- name: "Martin d'Anjou"
+  github: "martinda"
+- name: "Kristin Whetstone"
```

Comment:
GitHub IDs are needed at least to have good hyperlinks

Reply:
@oleg-nenashev I transcribed what @martinda wrote in the proposal; could you comment in Google Doc? Thanks.